docs/UserGuide.md

```diff
@@ -194,12 +326,15 @@ _Details coming soon ..._
 
 ## Command summary
 
-Action     | Format, Examples
------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------
-**Add**    | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
-**Clear**  | `clear`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit**   | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find**   | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List**   | `list`
-**Help**   | `help`
+| Action           | Format, Examples                                                                                                                                                      |
+|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**          | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague` |
+| **Clear**        | `clear`                                                                                                                                                               |
+| **Delete**       | `delete INDEX`<br> e.g., `delete 3`                                                                                                                                   |
+| **Edit**         | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`                                           |
+| **Find**         | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`                                                                                                            |
+| **Add Grade**    | `grade INDEX TEST_NAME SCORE WEIGHTAGE` <br> e.g., `grade 1 Midterm 85 30`                                                                                            |
+| **Delete Grade** | `deleteGrade INDEX TEST_NAME` <br> e.g., `deleteGrade 1 Midterm`                                                                                                      |
+| **Edit Grade**   | Simply add a new grade with the same test name, and the grade will be updated automatically. <br> e.g., `grade 1 Midterm 90 30`                                       |
```

Comment:
Does this belong in the command summary? The "edit" function is not a distinct command itself, so I feel like it might be confusing to place it in the summary table. Do you think [line 264](https://github.com/AY2425S1-CS2103T-T08-4/tp/pull/61/files#diff-b50feaf9240709b6b02fb9584696b012c2a69feeba89e409952cc2f401f373fbR264) is sufficient to explain how a user can perform edits?

Reply:
I think it is ok to leave edit grade here so users can quickly see how to edit a grade rather than go through all the texts, what do the rest think? @juancarlovieri @izruff @hpcman

Reply:
Should be fine, or we can use Add/Edit Grade in line 336

Reply:
I think writing Add/Edit on line 336 is better, since we probably will also write on the Add command section about how it can function as an Edit too.